Userland/Utilities/gron.cpp

```diff
@@ -41,15 +42,23 @@ int main(int argc, char** argv)
         return 1;
     }
 
-    if (argc != 2 || !strcmp(argv[1], "--help")) {
-        warnln("usage: gron <file>");
-        warnln("Print each value in a JSON file with its fully expanded key.");
-        return argc != 2 ? 1 : 0;
-    }
-    auto file = Core::File::construct(argv[1]);
-    if (!file->open(Core::OpenMode::ReadOnly)) {
-        warnln("Failed to open {}: {}", file->name(), file->error_string());
-        return 1;
+    Core::ArgsParser args_parser;
```

Comment:
You could also add the following here, so we don't remove the informational line above:
```cpp
args_parser.set_general_help("Print each value in a JSON file with its fully expanded key.");
```